cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -179,6 +179,7 @@ private boolean initAdminClient() {
             try {
                 this.allClient = adminClient(IntStream.range(0, podList.size()).boxed().collect(Collectors.toList()), false);
             } catch (ForceableProblem | FatalProblem e) {
+                LOGGER.infoCr(reconciliation, "Failed to create adminClient.", e);
```

Comment:
Should it be logged as a warning message if it as an error which we do not expect to happen?

Reply:
OK, updated. Thanks.